### PR TITLE
PYIC 3115: Create the putContraIndicators stub lambda with test

### PR DESCRIPTION
**Pull request by TalatAydinGDS**
### What changed

Created the putContraIndicators lambda in cimit-stub

### Why did it change

Create the putContraIndicators stub lambda with test

### Issue tracking

- [PYIC-3115](https://govukverify.atlassian.net/browse/PYIC-3115)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[PYIC-3115]: https://govukverify.atlassian.net/browse/PYIC-3115?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: di-ipv-cimit-stub/lambdas/put-contra-indicator-credential/src/main/java/uk/gov/di/ipv/core/putcontraindicatorcredential/PutContraIndicatorsHandler.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+public class PutContraIndicatorsHandler
+        implements RequestHandler<PutContraIndicatorsRequest, String> {
+
+    private static final Logger LOGGER = LogManager.getLogger();
+
+    @Override
+    public String handleRequest(PutContraIndicatorsRequest event, Context context) {
+        LOGGER.info(new StringMapMessage().with("EVENT TYPE:", event.getClass().toString()));
+        return "Success";
```
Comment (therealslimsladey) [inline di-ipv-cimit-stub/lambdas/put-contra-indicator-credential/src/main/java/uk/gov/di/ipv/core/putcontraindicatorcredential/PutContraIndicatorsHandler.java]: Return should be the json object 
```
{
  "result": "success"
}
```
Same with `PostMitigationsHandler`

File: di-ipv-cimit-stub/lambdas/put-contra-indicator-credential/build.gradle
Comment (therealslimsladey) [inline di-ipv-cimit-stub/lambdas/put-contra-indicator-credential/build.gradle:1]: Please move from `put-contra-indicator-credential` to `put-contra-indicators` for consistency with the CIMIT lambda naming.
Same for the java package.

**Review by therealslimsladey — CHANGES_REQUESTED** [review]

File: di-ipv-cimit-stub/lambdas/put-contra-indicator-credential/src/main/java/uk/gov/di/ipv/core/putcontraindicatorcredential/PutContraIndicatorsHandler.java (hunk 1 above)
Comment (TalatAydinGDS) [reply di-ipv-cimit-stub/lambdas/put-contra-indicator-credential/src/main/java/uk/gov/di/ipv/core/putcontraindicatorcredential/PutContraIndicatorsHandler.java]: Done.

File: di-ipv-cimit-stub/lambdas/put-contra-indicator-credential/build.gradle
Comment (TalatAydinGDS) [reply di-ipv-cimit-stub/lambdas/put-contra-indicator-credential/build.gradle:1]: Done.

File: di-ipv-cimit-stub/lambdas/put-contra-indicator-credential/src/main/java/uk/gov/di/ipv/core/putcontraindicatorcredential/PutContraIndicatorsHandler.java (hunk 1 above)
Comment (therealslimsladey) [reply di-ipv-cimit-stub/lambdas/put-contra-indicator-credential/src/main/java/uk/gov/di/ipv/core/putcontraindicatorcredential/PutContraIndicatorsHandler.java]: Sorry mate. I meant a JSON object with a "result" key that (for this iteration) takes the value "success"

File: di-ipv-cimit-stub/lambdas/put-contra-indicator-credential/build.gradle
Comment (therealslimsladey) [reply di-ipv-cimit-stub/lambdas/put-contra-indicator-credential/build.gradle:1]: The java package name as well?
`uk.gov.di.ipv.core.putcontraindicatorcredential` -> `uk.gov.di.ipv.core.putcontraindicators`

**Review by therealslimsladey — APPROVED** [review]
lgtm

**Pull request merged**